chia/full_node/coin_store.py

```diff
@@ -242,6 +242,22 @@ async def get_coins_removed_at_height(self, height: uint32) -> List[CoinRecord]:
                         coins.append(coin_record)
                 return coins
 
+    async def get_all_coins(self, include_spent_coins: bool) -> List[CoinRecord]:
+        # this is used by the simulator, using this normally would be a joke.
```

Comment:
can we elaborate this a bit like `WARNING: this should only be used for testing or simulation, running it on a synced testnet or mainnet node will most likely result in OOM`

Reply:
yeah that would probably be better :)